Sources/MapboxNavigation/Route.swift

```diff
@@ -42,4 +42,62 @@ extension Route {
             return LineString(trimmedPrecedingCoordinates + followingPolyline.trimmed(from: followingPolyline.coordinates[0], distance: distance)!.coordinates.suffix(from: 1))
         }
     }
+
+    var tollIntersections: [Intersection]? {
+        let allSteps = legs.flatMap { return $0.steps }
```

Comment:
There are already no return statements in small closures in this file, maybe it's better to remove them here?